src/System.Windows.Forms/src/System/Windows/Forms/LinkLabel.Link.LinkLabelLinkAccessibleObject.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class LinkLabel
+    {
+        public partial class Link
+        {
+            internal class LinkLabelLinkAccessibleObject : AccessibleObject
+            {
+                private readonly LinkLabelAccessibleObject _linkLabelAccessibleObject;
+                private readonly LinkLabel _owningLinkLabel;
+                private readonly Link _ownerLink;
```

Comment:
To be consistent with the rest
```suggestion
                private readonly Link _owningLink;
```